SPEC.md

```diff
@@ -306,6 +306,10 @@ For invoking a callback URL:
 - Issue a POST request to the caller-provided URL.
 - Include any callback headers supplied in the originating StartOperation request, stripping away the `Nexus-Callback-`
   prefix.
+- Include the `Nexus-Operation-Id` header, `Nexus-Operation-Start-Time` and any `Nexus-Link` headers for resources
+  associated with this operation to support completing asynchronous operations before the response to StartOperation is
+  received. `Nexus-Operation-Start-Time` should be in a valid HTTP/1.1 format. If is omitted, the time the completion is
```

Comment:
Could you clarify and possibly link here? https://www.rfc-editor.org/rfc/rfc5322.html#section-3.3